server/daos/booking_dao.js

```diff
@@ -0,0 +1,70 @@
+// import database
+// import modules
+
+const db = require('../db/db.js');
+const Booking = require('../models/booking.js');
+
+const createBooking = function (row){
+    return new Booking(row.lecture_id,row.student_id,row.waiting,row.present,row.updated_at,row.deleted_at);
+}
+
+// it creates the booking table
+exports.createBookingTable = function() {
+    return new Promise ((resolve,reject) => {
+        const sql = `CREATE TABLE Bookings (lecture_id TEXT NOT NULL, student_id TEXT NOT NULL,
```

Comment:
I think that `student_id` should be a NUMBER value

Reply:
Both lecture_id and student_id should actually be number

Reply:
Both lecture_id and student_id should actually be integers

Reply:
Yes you're right! Thanks